openquake/baselib/zeromq.py

```diff
@@ -76,7 +76,7 @@ class Socket(object):
     :param mode: default 'bind', accepts also 'connect'
     :param timeout: default 15000 ms, used when polling the underlying socket
     """
-    def __init__(self, end_point, socket_type, mode, timeout=15000):
+    def __init__(self, end_point, socket_type, mode, timeout=300):
```

Comment:
@vot4anto restore the timeout as it was!

Reply:
Yes, just do it with new PR with first files yaml of k8s